test/ItemRevUnitTests.hs.orig

```diff
@@ -0,0 +1,81 @@
+module ItemRevUnitTests (itemRevUnitTests) where
+
+import Prelude ()
+
+import Game.LambdaHack.Core.Prelude
+
+import qualified Control.Monad.Trans.State.Strict as St
+import qualified Data.EnumMap.Strict as EM
+import qualified Data.EnumSet as ES
+import qualified Data.Vector.Unboxed as U
+import qualified System.Random.SplitMix32 as SM
+
+import Test.Tasty
+import Test.Tasty.HUnit
+
+<<<<<<< HEAD
```

Comment:
WTF

Reply:
did you create this `hs.orig` file manually or is it a leftover from a non-finished rebase?